wix-inbox-v2/wix-inbox-v2/guides/EDM-intro.md

```diff
@@ -36,21 +33,15 @@ With the Inbox APIs, you can:
 
 <!-- - Handle webhooks when messages are sent to [a visitor](https://dev.wix.com/api/rest/inbox/messages/message-sent-to-participant-webhook) or [the business](https://dev.wix.com/api/rest/inbox/messages/message-sent-to-business-webhook), when a [message button is clicked](https://dev.wix.com/api/rest/inbox/messages/button-interacted-webhook), or when [conversations are merged](https://dev.wix.com/api/rest/inbox/conversations/conversations-merged-webhook). -->
 
-[Read more about how site contributors work with Wix Inbox][kb-inbox].
+[Learn more about how site collaborators work with Wix Inbox][kb-inbox].
 
 ## Terminology
 
-- A **message** is sent from the business to the visitor
-  or from the visitor to the business.
-  Messages can be one of these types: `basic`, `minimal`, `template`, `form`, `system`.
-  [Learn more about message types.][message-types]
-- A **conversation** is a collection of messages that takes place between two or more participants.
-  Conversations are displayed in the site's [Inbox][inbox-deeplink],
-  one conversation per site visitor, contact, or member.
-- A **participant** is the site visitor, contact, or member.
-  [Learn more about types of participant IDs.][visitor-id-types]
+- **Message:** A communication sent from the business to the visitor or from the visitor to the business.
+  Messages can be of various types, including `basic`, `minimal`, `template`, `form`, or `system` messages.
+  [Learn more about message types.](messages/message-types)
+- **Conversation:**  A collection of messages that takes place between businesses and participants. Conversations are displayed in the site's [Inbox][inbox-deeplink]. There can only be one conversation per site visitor, contact, or member.
```

Comment:
Ahh here's the difference between them. To me, this is a very important definition and it seems squished in here. I would maybe highlight the different submodules and what they do in the "With the Inbox APIs, you can:" section. And then link to the different submodules.